### PR TITLE
Minor doc fixes

### PR DESCRIPTION
**Pull request by ralexe**

### CODE REVIEW

File: src/debugger/revdebug.md [hunk 1]
```diff
@@ -101,5 +101,5 @@ And step back by `aesb`:
 
 In addition to the native reverse debugging capabilities in rizin, it's also possible to
 use gdb's remote protocol to reverse debug a target gdbserver that supports it.
-`=!dsb` and `=!dcb` are available as `dsb` and `dcb` replacementments for this purpose,
+`=!dsb` and `=!dcb` are available as `dsb` and `dcb` replacement's for this purpose,
```
Comment (Surendrajat) [inline src/debugger/revdebug.md:104]:
```suggestion
`=!dsb` and `=!dcb` are available as `dsb` and `dcb` replacements for this purpose,
```

Comment (Surendrajat) [reply src/debugger/revdebug.md:104]: Apostrophe needed?

**Review by Surendrajat — APPROVED** [review]

File: src/debugger/revdebug.md (hunk 1 above)
Comment (ralexe) [reply src/debugger/revdebug.md]: It's not needed. Thanks.

**Comment by Surendrajat** [comment]
Nice work. Thanks!

**Review by officialcjunior — APPROVED** [review]

**Review by wargio — APPROVED** [review]
Good job!

**Comment by wargio** [comment]
Thank you for the PR!

**Comment by officialcjunior** [comment]
Indeed. Awesome work! ⚡

**Comment by ralexe** [comment]
Thank you too!

**Pull request merged**